README.md

```diff
@@ -40,6 +40,8 @@ $ brew install --cask mactex
 $ sudo tlmgr update --self
 ```
 
+The build system uses [latexmk](https://mg.readthedocs.io/latexmk.html) to generate PDF. It probably has been installed on your OS already. If not, please follow the [installation guide](https://mg.readthedocs.io/latexmk.html#installation).
```

Comment:
`latexmk` is not a build system. Instead, it is a Perl script for running LaTeX the correct number of times to resolve cross references.
You can say, "latexmk is required to generated PDF."

Reply:
Changed as requested!